CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased/Snapshot]
 
+## [1.5]
```

Comment:
The tag `[1.5]` needs to be specified (cf. end of file)

Reply:
would prefer to do this in the v1.6-snapshot branch as I would like to include the merge commit of this branch into master into the range of the v1.5 commits